### ISSUE TITLE
write_sheet() not working when logged in through Chromium

### ISSUE THREAD

**Issue by timothoms** [tidyverse/googlesheets4#171]
I came across an issue where the sheet_write() function fails when I'm logged into Google Drive through the Chromium browser at the same time (but don't have the google sheet in question open in the browser).  It does work as soon as I log out. Is this behavior expected?

**Comment by jennybc** [comment]
This really needs a full reprex, because I think there's a good chance the cause-and-effect hypothesis here is wrong. I don't see any means of crosstalk between the token and, therefore, the user being used by googlesheets4 and the token and user being used by Chromium.

I can say, from vast experience, that it's very easy to create a Sheet from R as user A and to be logged in in a browser as user B and wonder why user B can't view the Sheet. Any permission problem is much more likely to be due to this sort of snafu, i.e. there's a mismatch between user in R and user in browser.

**Comment by timothoms** [comment]
I tried to create a simple MWE, and it worked, i.e. I could not recreate the problem. In the original problem, I was authorized as the same user in R as I was logged in through Chromium. That said, I realized that the snafu is probably the following: while I'm the owner of the googlesheet in question, this googlesheet is inside a shared Google Drive folder owned by another user. I will just try to avoid this sort of complicated setup. Many thanks, and my apologies for wasting your time; I realize now that I should have asked this in a different forum, and not posted it as an issue here.
